### PR TITLE
ajk/sft only

### PR DESCRIPTION
**Pull request by ajkhattak**
1. PR removes stuff that is not directly related to soil freeze-thaw and should not live in this repo as they have their own repos such as CFE and SoilMoistureProfiles 
2. PET and AORC bmi code is kept here for now but will be removed in the future

### Addition
- Updated REAMDE
- configuration, setting up external models (cloning other repos) are provided in the README
## Removals
- cfe code
- soil moisture code

## Testing

1. Tested in the pseudo-framework on two different machines (Rachel and myself)
2. Unchanged results before and after the changes

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [ x] Code can be automatically merged (no conflicts)
- [x] Code follows project standards (link if applicable)
- [x] Passes all existing automated tests
- [x ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,21 +20,22 @@ set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0")
 
 # add the executable
 
-## cfe + aorc + ftm
-#add_executable(cfeft ./src/main_pass_forcings.cxx ./src/cfe.c ./src/bmi_cfe.c ./forcing_code/src/aorc.c ./forcing_code/src/bmi_aorc.c)
+## cfe + aorc + pet + ftm
+add_executable(cfeft ./src/main_cfe_aorc_pet_ftm.cxx ./cfe/src/cfe.c ./cfe/src/bmi_cfe.c ./cfe/forcing_code/src/aorc.c ./cfe/forcing_code/src/bmi_aorc.c ./cfe/forcing_code/src/pet.c ./cfe/forcing_code/src/bmi_pet.c)
```
Comment (rlmcdaniel) [inline CMakeLists.txt:24]: Is the path to the first one correct (i.e. ./src/main_cfe_pet_ftm.cxx vs ./**cfe**/src/main_cfe_aorc_pet/ftm.cxx)?

File: README.md [hunk 2]
```diff
@@ -1,75 +1,50 @@
...
+
+<img width="607" alt="eq4" src="https://user-images.githubusercontent.com/15165757/157316409-3c51afda-e64b-4f6b-bf9e-efa50466cae4.png">	
+
+where T and T0  are the soil temperature and freezing temperatures, respectively (Ref), g is the acceleration of gravity [m s-2] and Lf is the latent heat of fusion [J Kg-1]. The soil matric potential,  [m], as a function of soil moisture content is given by (ref: Clapp-Hornberger):
+
+<img width="594" alt="eq5" src="https://user-images.githubusercontent.com/15165757/157317205-a2b1e455-8714-4113-8c56-f7a2ef55533c.png">
+
+where s, s, and l are saturated soil matric potential [m], saturated soil moisture [-] (porosity), and soil liquid moisture [-], respectively. The exponent b is the Clapp-Hornberger parameter. According to the ‘freezing equals drying’ approximation (Ref), we equate Eq. (4) and Eq. (5) to obtain an expression for liquid water content:
```
Comment (rlmcdaniel) [inline README.md:43]: psi-s and theta-s -> not sure if you can insert the symbols, but need to specify in some way.

File: configs/laramie_bmi_config_coupler.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+soil_params.smcmax=0.439
```
Comment (rlmcdaniel) [inline configs/laramie_bmi_config_coupler.txt:1]: Do we need to add units to this config file similar to the cfe config? [units]

File: configs/laramie_bmi_config_ftm.txt [hunk 4]
```diff
@@ -4,13 +4,16 @@ end_time_d=1035.91
 dt_s=3600.0
```
Comment (rlmcdaniel) [inline configs/laramie_bmi_config_ftm.txt:4]: Same question here: Do we need to add units to this config file similar to the cfe config? [units]

File: configs/laramie_bmi_config_smc_coupler.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+soil_params.smcmax=0.439
```
Comment (rlmcdaniel) [inline configs/laramie_bmi_config_smc_coupler.txt:1]: Same question here: Do we need to add units to this config file similar to the cfe config? [units]

File: configs/laramie_bmi_config_ftm.txt [hunk 6]
```diff
@@ -4,13 +4,16 @@ end_time_d=1035.91
...
 surface_temperature_bc=260
-#ice_fraction_scheme=Schaake
-ice_fraction_scheme=BMI
+ice_fraction_scheme=Schaake
+#ice_fraction_scheme=BMI
 Z=0.1,0.5,1.0,2.0
 soil_temperature=280.15,280.15,280.15,280.15
-#soil_total_moisture_content=0.389,0.396,0.397,0.397
-#soil_liquid_moisture_content=0.389,0.396,0.397,0.397
+#Z=0.1,0.5,1.0,2.0,3.0,4.0,5.0,6.0
```
Comment (rlmcdaniel) [inline configs/laramie_bmi_config_ftm.txt:14]: Delete these commented out soil layer (Z) and soil temp (soil_temperature) options.  They clutter the config file.  I think it's ok to leave the commented out ice_fraction_scheme to show the different options until we have more detailed documentation with that information.

File: CMakeLists.txt (hunk 1 above)
Comment (ajkhattak) [reply CMakeLists.txt:24]: yes, the pass is correct.

Comment (ajkhattak) [reply CMakeLists.txt:24]: *path

File: README.md (hunk 2 above)
Comment (ajkhattak) [reply README.md:43]: I corrected them with a trick, but I am not aware of any tool that can help write proper symbols here

File: configs/laramie_bmi_config_coupler.txt (hunk 3 above)
Comment (ajkhattak) [reply configs/laramie_bmi_config_coupler.txt:1]: yep, I have added units to the input parameters.

File: configs/laramie_bmi_config_ftm.txt (hunk 4 above)
Comment (ajkhattak) [reply configs/laramie_bmi_config_ftm.txt]: done

File: configs/laramie_bmi_config_smc_coupler.txt (hunk 5 above)
Comment (ajkhattak) [reply configs/laramie_bmi_config_smc_coupler.txt:1]: done

File: configs/laramie_bmi_config_ftm.txt (hunk 6 above)
Comment (ajkhattak) [reply configs/laramie_bmi_config_ftm.txt]: deleted

File: src/main_cfe_aorc_pet_ftm.cxx [hunk 7]
```diff
@@ -231,7 +269,7 @@ int
     4. Getting PET from PET and setting for CFE
     5. Get ice fraction from freeze-thaw model
     5. Update the CFE model.
```
[the remark this answers is not in the document]
Comment (ajkhattak) [reply src/main_cfe_aorc_pet_ftm.cxx]: done

**Review by rlmcdaniel — APPROVED** [review]

**Pull request merged**